pkg/keys/constants.go

```diff
@@ -133,6 +133,11 @@ var (
 	// LocalRangeLastReplicaGCTimestampSuffix is the suffix for a range's last
 	// replica GC timestamp (for GC of old replicas).
 	LocalRangeLastReplicaGCTimestampSuffix = []byte("rlrt")
+	// LocalRangeRangeTombstoneStatsUpdateLockSuffix is the suffix for a lock
+	// obtained by range tombstone operations to ensure they don't overlap with
+	// potentially long running GC queues while allowing point traffic to go
+	// through unobstructed.
+	LocalRangeRangeTombstoneStatsUpdateLockSuffix = []byte("rltu")
```

Comment:
`LocalRangeMVCCRangeKeyGCLockSuffix`.

This is only for serializing with GC, not for serializing stats updates in general (that happens through normal write latches, e.g. for two overlapping `MVCCDeleteRange` requests).